src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -79,8 +79,8 @@ export interface BalanceSummaryProps {
 }
 
 export default function BalanceSummary({ tokenAmount, nativeCurrencyAmount, isNative }: BalanceSummaryProps) {
-  const balanceUsdValue = useStablecoinValue(tokenAmount)?.toFixed(2)
-  const nativeBalanceUsdValue = useStablecoinValue(nativeCurrencyAmount)?.toFixed(2)
+  const balanceUsdValue = useStablecoinValue(tokenAmount)?.toFixed(6)
```

Comment:
i see .toFixed(6) in this file and the below - could you combine into one function?

Reply:
not sure how this can be combined... the only thing the function would be .toFixed(6). seems pointless to do it in a separate fn?

Reply:
added new fn actually, for more precision: currencyAmountToPreciseFloat